evm/15_core-concepts/46_EOAs-vs-Smart-Contracts.md

```diff
@@ -0,0 +1,85 @@
+---
+title: EOAs vs Smart Contracts
+contributors:
+  - { name: Esteban Saá, github: stebansaa }
+---
+  
+## EOAs vs Smart Contracts
+
+### What are EOAs?
+EOA stands for Externally Owned Account. In the context of EOSEVM, an EOA is an account that represents an individual user or entity on the EOSEVM network. It is similar to the concept of an Ethereum account. EOAs have their own addresses, private keys, and balances.
```

Comment:
All instances of `EOSEVM` should be `EOS EVM` to match the rest of the docs

Reply:
ok, will write the new one as part of arquitecture, if not just tell me where should it go.